src/main/java/com/hillel/items_exchange/dto/UserChangeEmailDto.java

```diff
@@ -18,10 +19,10 @@
 public class UserChangeEmailDto {
 
     @NotEmpty(message = "{invalid.not-empty}")
+    @Size(max = 129, message = "{too.big.email}")
```

Comment:
If I remember correctly we already discussed it and desided just to use Email annotation? @Jack11M , @drw85 do you remember such discussion?

Reply:
I'd like to discuss this.